src/server.c

```diff
@@ -3603,6 +3608,8 @@ int prepareForShutdown(int flags) {
         }
     }
 
+    aofManifestFree(server.aof_manifest);
```

Comment:
Here will crash in sentinel mode, it does not call `aofLoadManifestFromDisk`.

Reply:
fixed, thx.